lib/webpagetest.js

```diff
@@ -530,6 +530,202 @@ function runTest(what, options, callback) {
   return api.call(this, paths.test, callback, query, options);
 }
 
+function runTestAndWait(what, options, callback) {
+  delete options.pollResults;
+  delete options.timeout;
+  options = Object.assign(options, { pollResults: 10 });
+
+  var query = {};
```

Comment:
So all of this stuff is just the code that's in `runTest` right? Could we make this method basically a wrapper of that method instead to avoid duplication? So `runTestAndWait` basically just sets the pollResults option by default, calls `runTest` under the hood, and returns the result?

Reply:
Requested changes are done. Now, `runTestAndWait` acts as a wrapper around the `runtest` function (It now calls `runtest` under the hood) which assigns `pollResults ` as default in potions. Please review the updated code